src/main/java/sirius/biz/storage/layer2/BlobStorage.java

```diff
@@ -23,14 +26,31 @@
      * Determines if the space is browsable.
      * <p>
      * A browsable space has a root directory per tenant and can be viewed and used like a regular file system
-     * via the {@link L3Uplink}.
+     * via the {@link L3Uplink uplink}.
      */
     public static final String CONFIG_KEY_LAYER2_BROWSABLE = "browsable";
 
-    protected DerivedSpaceInfo<Boolean> editable = new DerivedSpaceInfo<>(CONFIG_KEY_LAYER2_BROWSABLE,
+    /**
+     * Determines if the space is mounted as readonly.
```

Comment:
I think this javadoc and the one on `CONFIG_KEY_LAYER2_BROWSABLE` is a bit misleading, as the constants dont determine anything on their own. The DerivedSpaceInfos below do that.